src/Modules/UngradedModules/FreeModuleHom.jl

```diff
@@ -517,17 +501,11 @@ julia> image(a)
 1 -> y*e[1]
 2 -> x*e[1] + y*e[2]
 3 -> z*e[2]
-represented as subquotient with no relations., Map with following data
-Domain:
-=======
-Submodule with 3 generators
+represented as subquotient with no relations., Hom:
```

Comment:
```suggestion
represented as subquotient with no relations, Hom:
```
We must kill off the full stop somewhere.

Reply:
done